app/Http/Controllers/ApiController.php

```diff
@@ -1,18 +1,19 @@
-<?php
-namespace App\Http\Controllers;
+<?php namespace App\Http\Controllers;
 use App\Http\Controllers\Auth\RegisterController;
 use App\User;
+
 use Illuminate\Http\Request;
 use App\Transaction;
 use Carbon\Carbon;
 use Illuminate\Support\Facades\Hash;
-
 /***
  * API Interface to Bet 360 app
  * Class ApiController
  * @package App\Http\Controllers
  */
 class ApiController extends Controller
+
+
```

Comment:
Opening brace of a class must be on the line following the class declaration; found 2 line(s)